### ISSUE TITLE
Keep stockpile stack coherency as you add/remove blocks

### ISSUE THREAD

**Issue by johnkslg** [StrangeLoopGames/EcoIssues#3562]
Better to have coherency than perfect groupings all the time.
![animx](https://user-images.githubusercontent.com/3536496/35189341-2c707824-fdfd-11e7-94ce-1ca44d4c598d.gif)

**Comment by metachronism** [comment]
I dont understand what you mean

**Comment by MarjieAVolk** [comment]
This annoys me a bit too.  It's like when you put 1 block into your stockpile, and then the whole stockpile contents gets completely shuffled around.  I imagine it would require quite a bit of tinkering to improve, thus why I put it on low priority.

**Comment by johnkslg** [comment]
Yes look at the animation, as I pull out a dirt it should only move that dirt, but every block in the pile moves. I realize this is to keep nice stacks, but I reckon we can try for both.

**Comment by metachronism** [comment]
@redwyre if you want to be the stockpile wizard you can experiment with a different grouping method
